requirements.txt

```diff
@@ -6,3 +6,4 @@ plotly==4.5.4
 jupyter==1.0.0
 simplejson==3.17.0
 boto3==1.12.28
+click==7.1.1
```

Comment:
This probably belongs in the next change w/ the `run.py` changes.

Reply:
ah yes. foreshadowing :)